discovery-provider/src/queries/get_remixable_tracks.py

```diff
@@ -68,5 +71,11 @@ def get_remixable_tracks(args):
 
         if args.get("with_users", False):
             add_users_to_tracks(session, tracks, current_user_id)
+        else:
```

Comment:
👍  we should think about the longer-term plan to make sure we don't have to remember do this, but TYSM for adding here!

Reply:
Yeah, my thinking is the longer term plan is "remove these `if` statements and always return users" ?

Reply:
👍